src/main/java/com/jcraft/jsch/jzlib/Deflate.java

```diff
@@ -1309,7 +1309,7 @@ int longest_match(int cur_match) {
           && window[++scan] == window[++match] && window[++scan] == window[++match]
           && scan < strend);
 
-      len = MAX_MATCH - strend - scan;
+      len = MAX_MATCH - (strend - scan);
```

Comment:
Can you add a comment here explaining that the parenthesis are required?
Otherwise someone may in the future believe they are unnecessary and remove them.

Reply:
No problem - I added a comment, based on my best understanding of the longest_match method.

Reply:
Can you please run `mvn formatter:format` so this comment is formatted correctly?

Reply:
In progress.

Reply:
Can you pull the third line of this comment up to the second line so that it doesn't appear to awkward?
I.e.
```
      // len here is set to MAX_MATCH minus (strend minus scan, i.e. the distance between scan and
      // strend) to ensure that best_len is set to the best possible length to lookahead
```